.github/workflows/createFB.yml

```
@@ -0,0 +1,58 @@
+name: Create Git Feature Branches for PLF projects 
+
+on:
+  workflow_dispatch:
+    inputs:
+      ORIGIN_BRANCH: 
+        type: text
+        description: like develop-meed or develop or develop-exo 
+        required: true
+      ISSUE: 
+        type: text
+        description: the task id 'task-1234' 
+        required: true     
+      BRANCH:
+        type: text
+        description: the feature branche you want to add it like mips devx ...
+        required: true
+      VALIDATE:
+        type: boolean
+        description: 'CHECK the box to PUSH'
+        default: false
+
+jobs:
+  create_fb:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+       - name: Checkout
+         uses: actions/checkout@v3
+         with: 
+           ref: ready
+           token: ${{ secrets.SWF_TOKEN }}
+       - name: install ssh
+         uses: shimataro/ssh-key-action@v2
+         with:
+           key: ${{ secrets.SSH_KEY }}
+           known_hosts: ${{ secrets.KNOWN_HOSTS }}.
```

```suggestion
           ssh-key: ${{ secrets.SSH_KEY }}
```
Since this scripts interacts to Github with ssh-key there is no need to include swf token